config/settings.yml

```diff
@@ -51,6 +51,9 @@ DOR_SERVICES:
   # To generate the token: docker-compose run dor-services-app rake generate_token
   TOKEN: eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJGb28ifQ.-BVfLTW9Q1_ZQEsGv4tuzGLs5rESN7LgdtEwUltnKv4
 
+PRESERVATION_CATALOG:
```

Comment:
Please lowercase these. The config gem documents the correct usage as lowercase: https://github.com/railsconfig/config#embedded-ruby-erb We have lowercase variables already in this file. If we use uppercase the config gem ends up creating methods with the ruby convention for constants, which is confusing.

Reply:
will do.  Actually, already done in PR #1603, and will rebase this afterwards.